File: client/src/app/employee/models/position.ts
import { EmployeePosition } from "./employee-position.module";

export class Position {

    id: number
    name: string
    employeePositions: EmployeePosition[];

    constructor(id: number, name: string, employeePositions: EmployeePosition[]) {
        this.id = id;
        this.name = name;
        this.employeePositions = employeePositions
    }

}

File: server/Practicum.Core/Models/Position.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Practicum.Core.Models
{
    public class Position
    {
        public int  Id { get; set; }
    
        public string Name { get; set; }
        public List<EmployeePosition> EmployeePositions { get; set; }
    }
}
